plugins/plugin-docker/che-plugin-docker-machine/src/test/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProviderTest.java

```diff
@@ -110,6 +113,9 @@
     @Captor
     private ArgumentCaptor<ContainerConfig> containerConfigArgumentCaptor;
 
+    @Mock
+    private RecipeRetriever recipeRetriever;
```

Comment:
Have you tested instance creation from both types of dockerfile - content and location?